android/codepush.gradle

```diff
@@ -2,8 +2,8 @@
 
 import java.nio.file.Paths;
 
-def config = project.hasProperty("react") ? project.react : [];
-def bundleAssetName = config.bundleAssetName ?: "index.android.bundle"
+def config = project.extensions.findByName("react") ?: []
+def bundleAssetName = config.bundleAssetName.get() ?: "index.android.bundle"
```

Comment:
* What went wrong:
A problem occurred evaluating script.
> No signature of method: java.util.ArrayList.get() is applicable for argument types: () values: []
  Possible solutions: get(int), get(int), set(int, java.lang.Object), set(int, java.lang.Object), grep(), grep()

Reply:
you're mixing Kotlin into gradle with .get()